server/routes/index.js

```diff
@@ -36,17 +43,32 @@ const createIndexRouter = ({ logger, hubFeaturedContentService, config }) => {
         postscript: true,
         detailsType: 'large',
         personalInformation,
-        userName: userName ? `Hi, ${userName}` : null,
         establishmentId,
         returnUrl: req.originalUrl,
       };
 
+      const data = {};
+
+      if (req.user) {
+        const userName = req.user && `Hi, ${req.user.getFullName()}`;
```

Comment:
`Hi, ` isn't part of their username, it's a display thing so should be separate. Either in the template or a separate variable.

Reply:
Actually, this would be better if we rename the var being passed in to `userNameDisplay` or something to indicate that?

Reply:
Resolved, updated to `welcomeMessage`